parser/src/python.lalrpop

```diff
@@ -387,7 +387,8 @@ MatchStatement: ast::Stmt = {
 }
 
 MatchCase: ast::MatchCase = {
-    <start:@L> "case" <pattern:Patterns> <guard:(Guard)?> ":" <body:Suite> <end:@R> => {
+    <start:@L> "case" <pattern:Patterns> <guard:(Guard)?> ":" <body:Suite> => {
+        let end = body.last().unwrap().end();
```

Comment:
Does body always exist? e.g. a match statement with empty body never join here?
In that case, adding a comment why unwrap never happens will be helpful.

Reply:
Yes, `body` always exists. `body` is not defined as optional and `Suite` only parses out a non-empty sequence of statements. 

https://github.com/RustPython/Parser/blob/4a2c4aad0be6c339dfd97437505f1fb35fe25f87/parser/src/python.lalrpop#L47-L53

This code is the same as for all other statements with a body

https://github.com/RustPython/Parser/blob/4a2c4aad0be6c339dfd97437505f1fb35fe25f87/parser/src/python.lalrpop#L879-L889

I'll add a comment